assets/js/custom/Program.js

```diff
@@ -460,8 +460,26 @@ const Program = function (projectId, csrfToken, userRole, myProgram, statusUrl,
       buttonIcon.hide()
     }
     buttonSpinner.removeClass('d-none')
+    console.log(url)
     window.location.href = url
   }
+
+  self.stealGame = function (url, programID) {
+    $.ajax({
+      url: url,
+      type: 'get',
```

Comment:
A GET request should not change something on the server. Take a look at the different methods :)

Reply:
I have used the "PUT" type for this ajax method because i just need to update and not change